iis-wf/iis-wf-export-actionmanager/src/test/java/eu/dnetlib/iis/wf/export/actionmanager/entity/EntityExportWorkflowTest.java

```diff
@@ -0,0 +1,25 @@
+package eu.dnetlib.iis.wf.export.actionmanager.entity;
+
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import eu.dnetlib.iis.common.AbstractOozieWorkflowTestCase;
+import eu.dnetlib.iis.common.IntegrationTest;
+
+/**
+ * @author mhorst
+ */
+@Category(IntegrationTest.class)
+public class EntityExportWorkflowTest extends AbstractOozieWorkflowTestCase {
+
+    @Test
+    public void testExportDatasetEntity() throws Exception {
+        testWorkflow("eu/dnetlib/iis/wf/export/actionmanager/entity/dataset/default");
```

Comment:
Generally you are right, the only problem is when it finally comes to defining second test for given module. Then it becomes rather painful because the only test defined until that time had to be moved into dedicated subfolder and all paths to resources and path to integration test had to be altered.